lib/nailed/config.rb

```diff
@@ -5,22 +5,29 @@
 module Nailed
   class Config
     class << self
+
+      # list of supported version-control-systems
+      @@supported_vcs = [ 'github' ]
+
       def parse_config
         is_valid?
 
+        # init class variables for supported VCS's
+        init_vcs
```

Comment:
The thing here is the `initialize` method would never be called, since there is no initialization with `Nailed::Config.new` anywhere in the code, due to the `class << self` in the `Config` class.
Calling a method from the `Config` class does not implicitly call `initialize`.

I could rename `init_vcs` to `initialize` but the method could still need to be called in `parse_config`, so I don't really see a point to do that in this case.

Reply:
Ok, then lets keep it that way.